app/public/technologies.json

```diff
@@ -1,464 +1,548 @@
 {
-  "git": {
-    "name": "git",
-    "ring": {
-      "name": "Develop",
-      "shape": "circle",
-      "color": "#9c27b0",
-      "startingPosition": {
-        "top": 0,
-        "left": 600
+   "TypeScript" : {
```

Comment:
Why the big changes?

Would expect only the added/modified lines.

Reply:
Possibly due to order change